app/src/main/res/drawable-v24/button_disabled.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:startColor="#954535"
```

Comment:
we should use colors, dimensions also along thing strings from resources
https://developer.android.com/guide/topics/resources/more-resources